x-pack/plugins/apm/public/components/app/ServiceDetails/MetricsChart.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiTitle } from '@elastic/eui';
+import React from 'react';
+import { GenericMetricsChart } from '../../../../server/lib/metrics/types';
+// @ts-ignore
+import CustomPlot from '../../shared/charts/CustomPlot';
+import { HoverXHandlers } from '../../shared/charts/SyncChartGroup';
+import { StringMap } from '../../../../typings/common';
+import { asPercent } from '../../../utils/formatters';
+import { Coordinate, YUnit } from '../../../../typings/timeseries';
+
+interface Props {
+  chart: GenericMetricsChart;
+  hoverXHandlers: HoverXHandlers;
+}
+
+function asBytesGB(value: number | null) {
+  if (value === null || isNaN(value)) {
+    return '';
+  }
+  return `${Math.floor(value / 1e7) / 100} GB`;
```

Comment:
Yeah I thought about this but wasn't sure if it would make sense to have it be that dynamic, but maybe you're right. Do we have this implemented already anywhere that you know of? If not I'll take a stab at a simple version.

Reply:
OK I added something but it might be problematic, will show you tomorrow.